IS7024FinalProjectBreweryArc/BreweryArc/.vs/BreweryArc/config/applicationhost.config

```diff
@@ -157,7 +157,7 @@
       </site>
       <site name="BreweryArc" id="2">
         <application path="/" applicationPool="BreweryArc AppPool">
-          <virtualDirectory path="/" physicalPath="C:\Users\parul\Documents\GitHub\IS7024-001\IS7024FinalProjectBreweryArc\BreweryArc" />
+          <virtualDirectory path="/" physicalPath="C:\Users\vaish\OneDrive\Documents\GitHub\IS7024-001\IS7024FinalProjectBreweryArc\BreweryArc" />
```

Comment:
Since this file has a path that is specific to the user's hard drive, it should be added to the .gitignore.